packages/node/src/indexer/dictionary/utils.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import { SubstrateCustomDatasource } from '@subql/types';
+
+export type SubstrateDsInterface = SubstrateCustomDatasource & {
```

Comment:
Why do we need this type defined? I would have thought we would have a type that already covers it